### PR TITLE
Update outdated json schema for #19

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: book/src/tech/dev/formats/scenarios.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ that show up in the UI. And of course, you can add as many people as you like.
     {
```
Comment (dabreegster) [inline book/src/tech/dev/formats/scenarios.md:32]: Could you please also update the description above, starting at line 16?

File: book/src/tech/dev/formats/scenarios.md [hunk 2]
```diff
@@ -14,25 +14,25 @@ It's easiest to follow the example below.
...
-The `mode` field could also be `Walk`, `Bike`, or `Transit`. The `purpose`
+The `mode` field can be be `Walk`, `Bike`, or `Transit`. The `purpose`
 field is mostly unused; you could pick [other
 values](https://a-b-street.github.io/abstreet/rustdoc/sim/enum.TripPurpose.html)
 that show up in the UI. And of course, you can add as many people as you like.
 
 ```
 {
-  "scenario_name": "monday",
+  "scenario_name": "minimal",
```
Comment (dabreegster) [inline book/src/tech/dev/formats/scenarios.md:30]: Yeah let's be real, nobody's Monday begins with midnight munchies in Montlake.

File: book/src/tech/dev/formats/scenarios.md [hunk 3]
```diff
@@ -79,6 +79,14 @@ To import this JSON file into A/B Street:
 cargo run --bin import_traffic -- --map=data/system/us/seattle/maps/montlake.bin --input=/path/to/input.json
 ```
 
+Then run the game as follows:
+
+```
+cargo run --bin game -- --dev data/system/us/seattle/maps/montlake.bin
+```
+
+You will need to select the senario with the user interface.
```
Comment (dabreegster) [inline book/src/tech/dev/formats/scenarios.md:88]: Or... `data/system/us/seattle/scenarios/montlake/minimal.bin`. Both work, of course.

File: book/src/tech/dev/formats/scenarios.md [hunk 4]
```diff
@@ -87,6 +95,8 @@ The tool will fail if any point doesn't match to a building. If you pass
 There are also a few tools that produce this JSON file:
 
 - <https://github.com/a-b-street/abstreet/blob/master/headless/examples/generate_traffic.py>
+- <https://github.com/a-b-street/abstr>
```
Comment (dabreegster) [inline book/src/tech/dev/formats/scenarios.md:98]: -fireworks emoji-

**Review by dabreegster — APPROVED** [review]
Thank you for fixing this up!

**Comment by Robinlovelace** [comment]
Cheers for the review! If you bump me up the permissions I can merge...

image

**Comment by dabreegster** [comment]
Let's test it out -- I gave everybody on the entire org write permission

**Pull request merged**